package.json

```diff
@@ -42,13 +42,13 @@
     "@hapi/hapi": "^20.2.1",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
-    "commitizen": "^4.2.4",
+    "commitizen": "^4.3.0",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^7.7.0",
+    "eslint": "^8.33.0",
```

Comment:
`@hapi/eslint-config-hapi` and ``@hapi/eslint-plugin-hapi`  are deprecated and moved to `@hapi/eslint-plugin`; this should be part of this update for `eslint`